DEVELOPING.md

```diff
@@ -0,0 +1,43 @@
+# Development guides and tips  
+
+## Linting
+
+### Run `golangci-lint`
+
+Full list linters and config can be found in `.golangci.yml`
+
+- Consider running the linter locally before commit.  
+
+  ```shell
+  make golangci-lint
+  ```
+
+- Or run the following to allow `golangci-lint` for any auto fixes (if available by linter)
+  
+  ```shell
+  golangci-lint run --fix --config .golangci.yml
+  ```
+
+### Suppress `golangci-lint` errors
+
+If you would like to suppress a false positive, consider in-line syntax.
+
+e.g error for gosec
+
+```shell
+lca-cli/ops/ops.go:162:17: G107: Potential HTTP request made with variable url (gosec)
+                        resp, err := http.Get(healthzEndpoint)
+```
+
+This can ignore with the following `//nolint:mylinter1,mylinter2`
+
+```go
+// ops.go L162 - suppress only gosec
```

Comment:
You're not suggesting a comment like this should be added, correct? Seems a little misleading to have this comment here

Reply:
Removed this comment from the code block